packages/block-editor/src/components/inserter/block-patterns-tab.js

```diff
@@ -214,7 +190,14 @@ export function BlockPatternsCategoryPanel( {
 		category.name === 'reusable'
 			? filteredUnsyncedPatterns
 			: currentCategoryPatterns;
-	const currentShownPatterns = useAsyncList( patterns );
+
```

Comment:
I did have a single unsynced list here but that caused a lot of problems with list placeholders being left in place when switching between unsynched and other categories